### PR TITLE
Add pages-config.json to README

### PR DESCRIPTION
**Pull request by mbland**
Could probably still use more love, and I need to get some configuration automation in-place, but it's a start.

cc: @konklone @gbinal @arowla

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,26 @@
...
-[Jekyll](http://jekyllrb.com/)-based [18F](https://18f.gsa.gov/) sites
-automatically from [GitHub](https://github.com/) repositories in a similar
-fashion to [GitHub pages](https://pages.github.com/).
+[18F Pages](https://pages.18f.gov/) is the platform that
+[18F](https://18f.gsa.gov/) uses to prototype and publish many of its
+[Jekyll](http://jekyllrb.com/)-based web sites. It works very similarly to
+[GitHub pages](https://pages.github.com/).
+
+This repo contains both the Jekyll source for
+[https://pages.18f.gov/](https://pages.18f.gov/) and the
```
Comment (arowla) [inline README.md]: Would be cleaner to just let pages URL get [autolinked](https://help.github.com/articles/github-flavored-markdown/#url-autolinking).

File: README.md [hunk 2]
```diff
@@ -121,6 +131,39 @@ location / {
...
+* **builders**: a list of individual webhook listeners/document publishers;
+  each item contains:
+  * **branch**: the branch from which to read document updates
+  * **repositoryDir**: the directory on the host machine into which all
+    repositories will be cloned
+  * **generatedSiteDir**: the directory on the host machine into which all
+    sites will be generated
+
+The **builders** list allows us to run one server to publish both
+[https://pages.18f.gov/](https://pages.18f.gov/) and the authenticated
```
Comment (arowla) [inline README.md]: More autolink candidates.

**Comment by arowla** [comment]
LGTM aside from the nitpicks.

**Comment by arowla** [comment]
Is there gonna be a blog post? :)

File: README.md (hunk 1 above)
Comment (mbland) [inline README.md]: Done.

File: README.md (hunk 2 above)
Comment (mbland) [inline README.md]: Done.

**Comment by mbland** [comment]
Heh, sure, I'd love to write a blog post! We actually have one in the works for @wslack's recent experience running through the Guides Template; figure we could roll this into that post, if it doesn't merit one of its own.

**Pull request merged**